src/InMemoryEventStore.php

```diff
@@ -151,6 +152,10 @@ public function load(
             }
         }
 
+        if (count($streamEvents) == 0) {
+            throw NoNewerEventsFound::with($streamName);
```

Comment:
@rabbl this exception catching can be removed completely. No StreamNotFound exception is thrown any more, when no newer events are found - an empty iterator is returned instead, so the catching can be removed. wanna PR this?